src/lib/inputUtils.ts

```diff
@@ -221,27 +221,10 @@ export async function validateUserMembership(snykMembership: {
   userEmail: string;
   role: string;
   org: string;
-}) {
+}, validRoles: any[]) {
```

Comment:
This util now requires an array of valid roles which will be passed from the group object

Reply:
Since the available roles are in the group context now, then lets move this function to the `snykGroup` class, and it will validate against the private member  `_roles`. Wdyt?